src/Skipper/SkipVoter/CommentSkipVoter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Skipper\SkipVoter;
+
+use PhpParser\Node;
+use Rector\Skipper\Contract\SkipVoterInterface;
+use Rector\Skipper\Skipper\SkipSkipper;
+
+final readonly class CommentSkipVoter implements SkipVoterInterface
```

Comment:
use a dedicated service instead, and use on `Skipper::shouldSkipCurrentNode()` to avoid bc break due to new `?Node $node` parameter addition.